doc/source/howtos/upgrade_guide.rst

```diff
@@ -42,7 +42,27 @@ If downtime is acceptable, shut down the cluster and upgrade all nodes to |pxc|
 
 If downtime is not possible, the rolling upgrade is supported but ensure the
 traffic is controlled during the upgrade and writes are directed only to 5.7
-nodes until all nodes are upgraded to 8.0.
+nodes until all nodes are upgraded to |version|.
+
+With |pxc| 5.7 nodes in the cluster, start |pxc| |version| nodes with
+``super_read_only`` enabled.
+
+.. seealso::
+
+   |MySQL| Documentation: super_read_only system variable
+      https://dev.mysql.com/doc/refman/8.0/en/server-system-variables.html#sysvar_super_read_only
+
+.. important:: 
+
+   If writes are directed to a |pxc| 8.0 node in a cluster that has |pxc| 5.7,
+   the cluster will be broken down. In this case, the system reports the
+   following error message:
```

Comment:
The message can vary depending on the query. So I would adjust the working to something like:

In this case, the system reports a message like below: